src/models.py

```diff
@@ -0,0 +1,67 @@
+import datetime
+from sqlalchemy import Column, Integer, String, DateTime, ForeignKey, BigInteger, DECIMAL
+from sqlalchemy.orm import relationship, backref
+from sqlalchemy.ext.declarative import declarative_base
+
+Base = declarative_base()
+
+# class Dummy(Base):
+#    __tablename__ = "dummy"
+#    id = Column(Integer, primary_key=True, index=True)
+#    dummy = Column(String(250),)
+
+
+class Currency(Base):
+    __tablename__ = "currency"
+
+    id = Column(Integer, primary_key=True, index=True)
+    symbol = Column(String(4), nullable=False, unique=True)
```

Comment:
We need to increase the symbol length since it can be longer, for example _BCHABC_ or _BURGER_.
I'd suggest something like 10 to avoid future problems.
```suggestion
    symbol = Column(String(10), nullable=False, unique=True)
```